Tests/MapboxMapsTests/Style/StyleTransitionTests.swift

```diff
@@ -0,0 +1,40 @@
+import XCTest
+
+#if canImport(MapboxMaps)
+@testable import MapboxMaps
+#else
+@testable import MapboxMapsStyle
+#endif
+
+//swiftlint:disable explicit_top_level_acl explicit_acl
+class StyleTransitionTests: XCTestCase {
+
+    func testDecodeHasCorrectConversion() {
+        let jsonString = """
+        {
+            "duration": 1000,
+            "delay": 500
+        }
+        """
+
+        if let jsonData = jsonString.data(using: .utf8) {
```

Comment:
I'd just force unwrap here since you control the input and you know that it doesn't fail.